### PR TITLE
implement onnx export for inception3/4, resnext, mobilenetv2

### PR DESCRIPTION
**Pull request by lostkevin**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

# Motivation
Please describe the motivation of this PR and the goal you want to achieve through this PR.

# Modification
Please briefly describe what modification is made in this PR.

# Backward-Compatibility-Breaking (Optional)
Does the modification introduce changes that break the backward-compatibility? If so, please describe how it breaks the compatibility in this PR.

# Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

# Pull-Request Todo List
1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: tools/export.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def main():
     cfg = mmcv_config_fromfile(args.config)
 
     if args.user_config_params is not None:
-        assert args.model_type is not None, 'model_type must be setted'
+        # assert args.model_type is not None, 'model_type must be setted'
```
Comment (Cathy0908) [inline tools/export.py:83]: 这一行为什么注释掉

File: configs/classification/imagenet/inception/inceptionv4_b32x8_100e.py [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+            loss_config=dict(
+                type='CrossEntropyLossWithLabelSmooth',
+                label_smooth=0,
+            ),
+            num_classes=num_classes,
+            input_feature_index=[0],
+        )
+    ])
+
+class_list = [
```
Comment (Cathy0908) [inline configs/classification/imagenet/inception/inceptionv4_b32x8_100e.py:33]: inception这几个配置文件可以提取一个base 配置吗，看起来重复度很高

Comment (lostkevin) [reply configs/classification/imagenet/inception/inceptionv4_b32x8_100e.py]: OK

File: tools/export.py (hunk 1 above)
Comment (lostkevin) [reply tools/export.py]: OK

**Pull request merged**